api/test/integration/run_tests.py

```diff
@@ -139,9 +129,7 @@ def run_test(test: str, iteration: int, pytest_mark: str = None):
             Extra mark used to pass the API integration test in a cluster or standalone environment.
         """
         test_name = \
-            f'{test.rsplit(".")[0]}' \
-            f'{STANDALONE_SUFFIX if pytest_mark == PYTEST_MARK_STANDALONE else CLUSTER_SUFFIX if pytest_mark == PYTEST_MARK_CLUSTER else ""}' \
-            f'{iteration if iteration != 1 else ""}'
+            f'{test.rsplit(".")[0]}' f'{iteration if iteration != 1 else ""}'
         html_params = [f"--html={RESULTS_FOLDER}/html_reports/{test_name}.html", '--self-contained-html']
         with open(os.path.join(RESULTS_FOLDER, test_name), 'w') as f:
```

Comment:
Review this block, `pytest_mark` is never used.

The `run_test` function was created to make the code clearer after the addition of the standalone mode. We could consider avoiding this function now. Have a look at the changes done on https://github.com/wazuh/wazuh/pull/12360.